### PR TITLE
fleshing out the outline and renaming paper

### PR DESCRIPTION
**Pull request by choldgraf**
Added a fair amount of content to the outline...I think we still need to work on structure etc, but we're getting there!

### CODE REVIEW

File: outline.md [hunk 1]
```diff
@@ -2,65 +2,149 @@
...
+
+## Improvements to be made
+### Technical and tooling improvements
+- Still areas of pain in technical course deployments
+- Why is there is a multiplicity of tools in this space?
+- NEED: Simpler pathway to deploying technical solutions for teaching (e.g., push-button jupyterhub)
+- NEED: More extensions and tooling around enhancing learning (e.g., interactivity and widgets)
+- NEED: More composable / swappable environment images (e.g., R and Python)
+- NEED: Academic-facing computational resources for teaching
+- NEED: Technology to facilitate data transfer and access for students
```
Comment (arokem) [inline outline.md:141]: We should add autoscaling in here

**Comment by choldgraf** [comment]
anything else to add other than autoscaling? I can take a pass so if you have other thoughts, lemme know!

**Comment by arokem** [comment]
Nothing comes to mind. Do you want to go ahead and merge this and start fleshing out sections? We can post a bunch of issues (one issue per section) and then split up the work between us on the different sections. At whatever resolution you think is appropriate.

**Comment by choldgraf** [comment]
Cool - let's merge and we can iterate from master from there

**Pull request merged**